source/winAPI/constants.py

```diff
@@ -0,0 +1,62 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2022 NV Access Limited
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+
+from enum import IntEnum
+
+
+class SystemMetrics(IntEnum):
+	"""
+	GetSystemMetrics constants
+	https://docs.microsoft.com/en-gb/windows/win32/api/winuser/nf-winuser-getsystemmetrics

```

Comment:
Since these are `winuser` constants, the values living in winUser makes makes them easier to find.
The options I see right now:
- Keep these in `winUser`, moving that to `winAPI` later.
- Add a new `winUser` to `winAPI` (containing only these constants) now, accepting that it will be confusing to have two identically named modules.
- Rename this to `winUserConstants`.

Reply:
As discussed, this has been moved to `winAPI.winUser.constants`